tests/src/performance/Scenario/JitBench/Benchmarks/EmptyConsoleProgramExecution.cs

```diff
@@ -0,0 +1,104 @@
+            const string sourceFile = "Program.cs";
+            const string csprojFile = "console.csproj";
+
+            string consoleProjectMainDir = GetJitBenchRepoRootDir(intermediateOutputDir);
+            FileTasks.DeleteDirectory(consoleProjectMainDir, output);
+            FileTasks.CreateDirectory(consoleProjectMainDir, output);
+
+            File.WriteAllLines(Path.Combine(consoleProjectMainDir, sourceFile), new[] {
+                "using System;",
+                "public static class Program",
+                "{",
+                "    public static int Main(string[] args) => 0;",
+                "}"
+            });
+
+            File.WriteAllLines(Path.Combine(consoleProjectMainDir, csprojFile), new[] {
+                @"<Project Sdk=""Microsoft.NET.Sdk"">",
+                @"  <PropertyGroup>",
+                @"    <OutputType>Exe</OutputType>",
+                @"    <TargetFramework>netcoreapp2.1</TargetFramework>",
+                @"  </PropertyGroup>",
+                @"</Project>",
+            });
+        }
+
+        private async Task<string> Publish(DotNetInstallation dotNetInstall, string outputDir, ITestOutputHelper output)
+        {
+            string tfm = DotNetSetup.GetTargetFrameworkMonikerForFrameworkVersion(dotNetInstall.FrameworkVersion);
+            string publishDir = GetAppPublishDirectory(dotNetInstall, outputDir, tfm);
+            if (publishDir != null)
+                FileTasks.DeleteDirectory(publishDir, output);
+
+            string dotNetExePath = dotNetInstall.DotNetExe;
+            await new ProcessRunner(dotNetExePath, $"publish -c Release -f {tfm}")
+                .WithWorkingDirectory(GetAppSrcDirectory(outputDir))
+                .WithEnvironmentVariable("DOTNET_MULTILEVEL_LOOKUP", "0")
+                .WithEnvironmentVariable("UseSharedCompilation", "false")
+                .WithLog(output)
+                .Run();
+
+            publishDir = GetAppPublishDirectory(dotNetInstall, outputDir, tfm);
+            if (publishDir == null)
+                throw new DirectoryNotFoundException("Could not find 'publish' directory");
+
+            ExePath = Path.Combine(publishDir, ExecutableName);
+            return publishDir;
+        }
+
+        private string GetAppPublishDirectory(DotNetInstallation dotNetInstall, string outputDir, string tfm)
+        {
+            string dir = Path.Combine(GetAppSrcDirectory(outputDir), "bin", dotNetInstall.Architecture, "Release", tfm, "publish");
+            if (Directory.Exists(dir))
+                return dir;
+
+            dir = Path.Combine(GetAppSrcDirectory(outputDir), "bin", "Release", tfm, "publish");
+            if (Directory.Exists(dir))
+                return dir;
+
+            return null;
+        }
+
+        private static string GetAppSrcDirectory(string outputDir) =>
+            Path.Combine(GetJitBenchRepoRootDir(outputDir));
+
+        private static string GetJitBenchRepoRootDir(string outputDir) =>
```

Comment:
Is there a reason we're calling this GetJitBenchRepoRootDir? We don't seem to pull down JitBench

Reply:
In this case/repo, `JitBench` is an overloaded term. It is used as the set of JIT benchmarks, and as the `MusicStore/JitBench` benchmark itself.

Reply:
But it isn't something that we use in all of the benchmarks. For example, in the Word2Vec scenario, we use GetWord2VecNetRepoRootDir: https://github.com/dotnet/coreclr/blob/master/tests/src/performance/Scenario/JitBench/Benchmarks/Word2VecBenchmark.cs#L209

I think we should probably change this to something that describes the benchmark we are running, especially since there is no "repo" in this instance.

Reply:
makes sense